chainsight-cdk/src/config/components.rs

```diff
@@ -138,6 +138,7 @@ pub struct RelayerConfig {
     pub oracle_type: String,
     pub method_identifier: String,
     pub abi_file_path: String,
+    pub method_name: Option<String>,
```

Comment:
In chainsight-cli, If Manifest is for Oracle, how about setting "update_state" from cli?
I think it's ok to use String instead of Option for this definition.